common/src/envs/constants.ts

```diff
@@ -246,7 +246,7 @@ export const GROUP_SLUGS_TO_IGNORE_IN_MARKETS_EMAIL = [
   'manifold-features-25bad7c7792e',
   'bugs',
   'manifold-leagues',
-  'nonpredictive',
```

Comment:
This is wrong, you're still using the nonpredictive slug here: https://manifold.markets/browse?topic=nonpredictive&t=&q=

Reply:
If you wanted to switch the group slugs you'd have to switch it on all the denormalized data on contracts, which would not be trivial.

Reply:
Yeah u right! Forgot I had made this change. Was planning to change the slug but then realised was easier not to and forgot to change this back. Good catch